### PR TITLE
Add python builds to the build matrix, build with newer compilers

### PR DESCRIPTION
**Pull request by shadowwalkersb**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,13 +10,13 @@ source:
 
 build:
     number: 17
```
Comment (mstabrin) [inline recipe/meta.yaml:12]: Should we increase the build number to make the distinguishing clear?

**Comment by mstabrin** [comment]
Do pydusa Pull Requests also get uploaded to anaconda automatically, i.e. is this version here the version that was uploaded as linux-64/pydusa-1.15-py37h414ee2b_17.tar.bz2 ?

File: recipe/meta.yaml (hunk 1 above)
Comment (shadowwalkersb) [reply recipe/meta.yaml:12]: Didn't want to update `eman-deps`, so kept the build number.

**Comment by shadowwalkersb** [comment]
> is this version here the version that was uploaded as linux-64/pydusa-1.15-py37h414ee2b_17.tar.bz2 ?

This is one of the five linux versions that was uploaded.

**Pull request merged**